python/test/unit/test_contexts.py

```diff
@@ -20,48 +20,72 @@
 """
 
 class Contexts(unittest.TestCase):
-    def test_default_context(self):
-        ctx = arb.context()
-
-    def test_resources(self):
+    def test_default_allocation(self):
         alloc = arb.proc_allocation()
 
-        # test that by default proc_allocation has 1 thread and no GPU, no MPI
+        # test that by default proc_allocation has 1 thread and no GPU
         self.assertEqual(alloc.threads, 1)
+        self.assertEqual(alloc.gpu_id, None)
         self.assertFalse(alloc.has_gpu)
-        self.assertEqual(alloc.gpu_id, -1)
 
+    def test_set_alloc(self):
```

Comment:
for consistency with the first test, use the full word `allocation` in the title, i.e. `test_set_allocation`. Same goes for the tests below.

Reply:
Done.